nengo_gui/guibackend.py

```diff
@@ -102,17 +103,47 @@ class GuiRequestHandler(server.HttpWsRequestHandler):
         '/favicon.ico': 'serve_favicon',
     }
 
-    def get_expected_origins(self):
-        session = self.get_session()
+    def _is_loopback(self, host):
```

Comment:
You're right. I wrote that comment when I was sleepy. I shouldn't review code when I'm sleepy.